### DISCUSSION TITLE
Buttons Randomly Not Skinned

### DISCUSSION THREAD

**Discussion by d87** [SFX-WoW/Masque#103] [Questions] [answered]
### Retail

Retail

### Game Version:

9.0.0

### Add-On Version

8.3.0

### Describe the Issue

Some buttons are just randomly bugged / not skinned
Using Dominos and this skin https://github.com/rgd87/Masque_Brownstone/releases
Majority of buttons are usually fine, /reload doesn't help, clean relog neither, same buttons stay bugged between reloads, But over a longer time there's no pattern. If you manually reselect skin all buttons are fixed until reload.
The issue is with the skin it seems because other skins i have are alright. But what could be the reason?

### Screenshots

![image](https://user-images.githubusercontent.com/343331/92603438-80be4680-f2d9-11ea-81bd-dc19d15b61e0.png)

Edit: oh yeah, also it seems playing on test client aggravates this issue, because it's not like it was always that way, On live was fine, on 8.3 PTR it was rarely but happening, in 9.0 it's all the time

**Comment by StormFX** [comment]
Is it possible that you've made some local changes that aren't reflected in the release on your project's page? I tested the skin and didn't see any issues after a dozen or so reloads. I might be able to figure out what's going on if I have a solid way to reproduce it, but as of now, I've no clue.

**Comment by d87** [comment, marked as answer]
Shit, you're right... I fixed it here https://github.com/rgd87/Masque_Brownstone/commit/ad847b0610e12e3b81ddc4dfe7ece17b25ab0a70
and totally forgot about it. and my repo was old

**Comment by StormFX** [comment]
Glad you figured it out.

**Comment by d87** [comment]
Thanks, although you still might want to take a look at that. Because it really wasn't obvious that i have to specify a template or whatever fixed it.
Like i said it was working great for a long time, and it still halfway does without a template. This was pretty crazy

**Comment by StormFX** [comment]
I've got plans to overhaul the wiki, but that's a bit off. As far as the template goes, that's only for when wants to reuse most of the settings from another skin. Otherwise, you'd need the code for the _entire_ skin.
